about/jobs/template.md

```diff
@@ -0,0 +1,81 @@
+
+4. In your summary, put the following:
+
+ >I'm currently on a full-time immersive web development course at the highly competitive Founders & Coders academy in Bethnal Green. By the end of the course I will be equipped with tools to build full-stack projects.
+
+ >We have a number of developers at Founders and Coders who are open to being contacted about vacancies or contracting opportunities. You can email us here:
+
+ >hireus@foundersandcoders.com
+
+ >We are also open to hearing about projects you may be interested in being built. Get in touch on the same e-mail for any web development work you need. We'd love to hear from you!
+
+ Also link to your GitHub profile in your summary! (press the **link** button when you're editing your profile)
+
+5. In the Founders and Coders section of your experience, add the following:
+
+  >A four month intensive web development bootcamp, covering the following and more:
+
+ >-HTML, CSS & Javascript   
+ >-Git and GitHub  
+ >-Node.js  
+ >-Agile and Test-Driven Development   
+
+ Then add a link to the website as well as a link to the Guardian article or any other of the number of articles FAC has featured in. Spread the word!
+
+6. Respond to recruiters. We _really_ want to make sure we all have a number of opportunities to look at if any of us consider leaving FAC for a full-time job. It is important, therefore, for FAC to forge a number of direct and sustainable relationships with the appropriate companies. This can be achieved quicker and in greater numbers if we all respond to direct contact we get. Please see the __template message__ below
+
+7. Don't put your personal email or number on your profile
+
+##### Quick note on recruiters
+
+You will be contacted either by an internal recruiter :smile: or an external recruiter :unamused: (which I used to be):
+
+* __Internal recruiter__ - the ones we're most interested in. They work for the company where the vacancy is and therefore *normally* have the interests of the company at heart, and will be in contact with the hiring managers __far more often__ than the external recruiter will be. Why make the process slower by including a middle man? They will also be _far_ more likely to understand the culture of the company as well as understand how _well_ any of us fit the opportunity
+
+* __External recruiter__ - they're tricky. They work almost *entirely* on commission so _all_ they want to do is find someone who can pass an interview, get a job and work long enough for them to claim the recruitment fee. We *may* be interested in them if they can split a recruitment fee but this is not *particularly* likely. You could try speaking with them to find out the name of the company they are working with and then we can get in direct contact. This really **isn't** the most noble thing to do, but our cause is noble in sustaining the FAC community and academy (do you think the recruiter will be doing the same thing with his or her fee?)
```

Comment:
I would advise against advising people to do anything such as getting the company name out of the recruiters so we can contact them directly - you're right when you say it isn't right, so let's just take it out.